Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Footer.css";
+
+function Footer (){
+    return (
+        <div className="footer">
+            <i class="fal fa-comment"></i>
+            <i class="fal fa-retweet"></i>
+            <i class="fal fa-heart"></i>
+            <i class="fal fa-envelope"></i>
```

Comment:
Making the following changes should get your icons to render.
```suggestion
      <i className="far fa-comment" />
      <i className="fas fa-retweet" />
      <i className="far fa-heart" />
      <i className="far fa-envelope" />
```